include/mppic/critics/goal_angle_critic.hpp

```diff
@@ -26,7 +26,7 @@ class GoalAngleCritic : public CriticFunction
     nav2_core::GoalChecker * goal_checker) override;
 
 protected:
-  double threshold_to_consider_goal_angle_{0};
+  double distance_to_goal_upper_activation_threshold_{0};
```

Comment:
Is this in the wrong file? I'm not sure this will compile if its use wasn't changed in goal angle too?

Reply:
fixed. I'm not testing much this branch now. I wait refactoring branch to be merged to start testing this one

Reply:
Understoof